CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
```

Comment:
🏆

Reply:
I think it's great we're documenting these details in the CHANGELOG, good stuff 🌟